ci-operator/step-registry/cucushift/installer/check/nutanix/vip/cucushift-installer-check-nutanix-vip-commands.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+check_result=0
+# Ensure our UID, which is randomly generated, is in /etc/passwd. This is required
+# to be able to SSH.
+if ! whoami &> /dev/null; then
+    if [[ -w /etc/passwd ]]; then
+        echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+    else
+        echo "/etc/passwd is not writeable, and user matching this uid is not found."
+        exit 1
+    fi
+fi
+
+oc new-project vip-test
+oc new-app httpd -n vip-test
+oc expose svc/httpd
+oc scale deployment/httpd --replicas=3
+# Check all 3 pods was successfully deployed and running
+loop=10
+while [[ ${loop} -gt 0 ]]; do
+    ready_replicas=$(oc get deployment httpd -o jsonpath='{.status.readyReplicas}')
+    if [[ "${ready_replicas}" == 3 ]]; then
+        echo "Pass: check 3 pods was successfully deployed and running"
+        break
+    else
+        echo "Wait: not ready yet"
+        loop=$((loop - 1))
+        sleep 3
+    fi
+    if [[ ${loop} == 0 ]]; then
+        echo "Timeout: failed due to timeout"
+        exit 1
+    fi
+done
+
+# Check round-robin algorithm, each pod was deployed on a different worker
+ep_count=$(oc get ep httpd -o jsonpath='{.subsets[0].addresses}' | jq length)
+if [[ "${ep_count}" == 3 ]]; then
+    echo "Pass: check each pod was deployed on a different worker"
+else
+    echo "Fail: check each pod was deployed on a different worker"
+    check_result=$((check_result + 1))
+fi
+
+# Check ingressVIP works well
+route_host=$(oc get routes httpd -o jsonpath='{.spec.host}')
+if curl --connect-timeout 6 "$route_host"; then
+    echo "Pass: check ingressVIP works well"
+else
+    echo "Fail: check ingressVIP works well"
+    check_result=$((check_result + 1))
+fi
+
+# Check apiVIP works well
+set +e
+curl --connect-timeout 6 "$(oc get infrastructure cluster -o jsonpath='{.status.apiServerURL}')"
+if [[ $? == 60 ]]; then
```

Comment:
Compared with ingressVIP url check point, here we are using "60" to check a success return?

Per cur manual:
```
60     Peer certificate cannot be authenticated with known CA certificates
```
Sounds it is not a strict success, sounds like we can use "-k" for curl command to skip it.